scss/_dropdown.scss

```diff
@@ -32,19 +32,29 @@
   border: $dropdown-border-width solid $dropdown-border-color;
   @include border-radius($dropdown-border-radius);
   @include box-shadow($dropdown-box-shadow);
+
+  // Reset positioning when positioned with Popper
+  &[style] {
+    right: auto; // TODO RTL?
```

Comment:
This needs a check @ffoodd for RTL (and the whole PR)

Reply:
On the RTL side, this will be translated to `left: auto`.
If this part is meant to reset initial positionning, maybe we should reset both sides? If so, it'll need ` #{"/* rtl:ignore */"}` as seen a few lines below.